client/components/Coverages/coverage-icons.scss

```diff
@@ -0,0 +1,23 @@
+.coverages-popup {
+    background-color: var(--color-dropdown-menu-Bg);
+    border-radius: 5px;
```

Comment:
To provide a consistent and predictable interface, I would suggest we use variables defined in the ui-framework, including radius, padding etc.

* [border-radius: var(--b-radius--medium)](https://github.com/superdesk/superdesk-ui-framework/blob/develop/app/styles/design-tokens/_design-tokens-general.scss#L13)
* [sd-font-size--x-small](https://github.com/superdesk/superdesk-ui-framework/blob/develop/app/styles/_helpers.scss#L957)
* [Paddings & margins](https://ui-framework.superdesk.org/#/components/spacing)

I could not find any CSS var for `line-height`, so maybe changing to using `rem` instead of `px`.

All these are important to be able to support different themes for Superdesk. It may be a requirement for an accessibility theme to provide bigger margins, less rounded corners etc, and hard coding these values will not support this.

Reply:
I've updated it. I'd say we should use more variables separately rather than adding a long list of classes to HTML elements. One reason is to avoid repetition like in this file. Another is that working this way you can't see all style declarations in one place for a particular classname which makes working on it a lot slower.